cri/v1alpha2/cri.go

```diff
@@ -608,6 +614,10 @@ func (c *CriManager) PodSandboxStatus(ctx context.Context, r *runtime.PodSandbox
 		}
 	}
 
+	if v, exist := annotations[passthruKey]; exist && v == "true" {
+		ip = annotations[passthruIP]
```

Comment:
No need to check the existence of `passthruIP`?

Reply:
`passthruIP` can be empty string?